pkg/metrics/metrics.go

```diff
@@ -630,6 +641,7 @@ func DefaultMetrics() map[string]struct{} {
 		Namespace + "_policy_change_total":                                           {},
 		Namespace + "_policy_endpoint_enforcement_status":                            {},
 		Namespace + "_policy_implementation_delay":                                   {},
+		Namespace + "_cidr_group_policies":                                           {},
```

Comment:
This is leading to 

```
level=warning msg="Metric does not exist, skipping" metric=cilium_cidr_group_policies
```

in CI.

Looks like we have just a typo, `_cidrgroup_policies` vs `_cidr_group_policies`.